assets/backport/main.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/gravitational/trace"
+	"github.com/teleport/assets/backport/github"
+	"gopkg.in/yaml.v2"
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+
+	backportBranches, prNumber, owner, repo, err := parseFlags()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Getting the Github token from ~/.config/gh/hosts.yml
+	token, err := getGithubToken()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	clt, err := github.New(ctx, &github.Config{
+		Token:        token,
+		Repository:   repo,
+		Organization: owner,
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, targetBranch := range backportBranches {
+		// New branches will be in the format:
+		// auto-backport/[pull request number]-to-[target branch name].
+		newBranchName, err := clt.Backport(ctx, targetBranch, prNumber)
+		if err != nil {
+			log.Fatal(err)
+		}
+		fmt.Printf("Backported commits to branch %s.\n", newBranchName)
+
+		// Create the pull request.
+		if err = clt.CreatePullRequest(ctx, targetBranch, newBranchName, prNumber); err != nil {
+			log.Fatal(err)
+		}
+		fmt.Printf("Pull request created for branch %s.\n", newBranchName)
+	}
+	fmt.Println("Backporting complete.")
+}
+
+type Config struct {
+	Host Host `yaml:"github.com"`
+}
+
+type Host struct {
+	Token string `yaml:"oauth_token"`
+}
+
+// githubConfigPath is the default config path
+// for the Github CLI tool.
+const githubConfigPath = ".config/gh/hosts.yml"
```

Comment:
Can you add a comment about this path is relative to - I assume user's home dir, but could easily be repo workspace, fs root, etc.